x-pack/test_serverless/api_integration/test_suites/observability/dataset_quality_api_integration/utils/data_stream.ts

```diff
@@ -24,3 +25,35 @@ export async function getDataStreamSettingsOfEarliestIndex(es: Client, name: str
 
   return matchingIndexesObj[matchingIndexes[0]].settings;
 }
+
+function getCurrentDateFormatted() {
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+
+  return `${year}.${month}.${day}`;
+}
+
+export function getBackingIndexNameWithoutLastPart({
```

Comment:
I think this name is misleading, are we still getting the last part of the backing index, no?
Maybe we are just creating a new backingIndexName here?